examples/basic_bitcoin/test/tests.ts

```diff
@@ -37,15 +37,15 @@ export function getTests(canisterId: string): Test[] {
         {
             name: '/get-p2pkh-address',
             test: async () => {
-                const address = await getP2pkhAddress(origin);
+                const address = await getP2wpkhAddress(origin);
```

Comment:
This seems like an issue, the basic_bitcoin example should use p2pkh addresses. Also these tests should have failed, so let's investigate why they didn't.

Reply:
Done

Reply:
Tests did fail, code has been fixed